Update steps.css

Christopher-O opened the pull request:
-removed style for lead zero in striped,-shrunk min height for list and sub list to remove extra spacing when content is two lines or less

duboisp commented:
@Christopher-O prior to merge your change, we will need a little bit more such a revised implementation plan and to have tangible evidence that this experimentation are progressing toward a stable component. That evidence can be user research report, accessibility assessment, ACR, translation, proof that DTO completed (or started to draft) the guidance..... as it was specified in the [current implementation plan](https://wet-boew.github.io/GCWeb/m%C3%A9li-m%C3%A9lo/2021-05-steps/d%C3%A9tails-en.html)

I will be there at the [office hours](https://github.com/wet-boew/wet-boew/wiki/WET-Office-hours,-Heures-de-service-de-la-BOEW) to answer are your question.

duboisp commented:
as discussed,

* [ ] Create a page with the French translation of the working example
* [ ] Produce an accessbility assessment and upload it to the project experimental folder.
* [ ] Seperate the Working example vs the Guidance.
* [ ] Cross-link all the project pages and documents to ensure we can find all the information published via our GCWeb website.

Garneauma commented:
Pre-approved upon successful review and completion of all work items.

Christopher-O commented:
Waiting for @duboisp for conversion of WCAG accessibility points to HTML, 
unsure exactly what this task entails
"Cross-link all the project pages and documents to ensure we can find all the information published via our GCWeb website."
@Garneauma could you please advise?

duboisp commented on méli-mélo/2021-05-steps/steps-doc-en.html:
```diff
@@ -0,0 +1,134 @@
+  <section>
+    <h2>Working examples</h2>
+    <ul>
+      <li><a href="2021-05-steps.html">English example</a></li>
+      <li><a href="2021-05-steps-fr.html">French example</a></li>
```
Broken link

```suggestion
      <li><a href="index.html">English example</a></li>
      <li><a href="index-fr.html">French example</a></li>
```

duboisp commented on méli-mélo/2021-05-steps/steps-doc-fr.html:
```diff
@@ -0,0 +1,134 @@
+  <section>
+    <h2>Exemples pratiques</h2>
+    <ul>
+      <li><a href="2021-05-steps.html">Exemple anglais</a></li>
+      <li><a href="2021-05-steps-fr.html">Exemple français</a></li>
```
Broken links

```suggestion
      <li><a href="index.html">Exemple anglais</a></li>
      <li><a href="index-fr.html">Exemple français</a></li>
```

duboisp commented on méli-mélo/2021-05-steps/a11y-1-steps.json:
```diff
@@ -0,0 +1,651 @@
+        "dct:description": "Assess the GCWeb meli-melo steps page according to Standard on Web Accessibility (SWA)",
+        "earl:pointer": {
+            "@type": "oa:CssSelector",
+            "@value": "steps"
```
Must be a valid CSS selector, there is no element `<steps>`.

```suggestion
            "@value": ".lst-stps"
```

duboisp reviewed (changes requested):
@ouafaaetta please submit your change suggestion for the accessibility report like you did in the other PR. Thanks

I completed the code review with the local testing.  There is a few typo which we will apply prior to merge this change. 

Note: There is some additional change would be required, but those are not mandatory right now like:
* Using jekyll front-matter to specify the other page language
* Adding the 2 HTML placeholder page to render the accessibility assessment in an HTML format.
* Producing the index.json-ld file for documentation by leveraging the documentation provided.

ouafaaetta commented on méli-mélo/2021-05-steps/a11y-1-steps.json:
```diff
@@ -0,0 +1,651 @@
+                "earl:Assertion"
+            ]
+        },
+        {
+            "@id": "_:sc_3.2.2",
+            "earl:test": "WCAG21:on-input(Level A)",
+            "earl:subject": "_:subject",
+            "earl:outcome": "earl:inapplicable",
+            "dct:description": "As defined",
+            "earl:mode": "earl:semiAuto",
+            "@type": [
+                "earl:TestResult",
+                "earl:Assertion"
+            ]
+        },
```
```suggestion
        {
            "@id": "_:sc_3.2.2",
            "earl:test": "WCAG21:on-input",
            "earl:subject": "_:subject",
            "earl:outcome": "earl:inapplicable",
            "dct:description": "As defined",
            "earl:mode": "earl:manual",
            "@type": [
                "earl:TestResult",
                "earl:Assertion"
            ]
        },
```

ouafaaetta commented on méli-mélo/2021-05-steps/a11y-1-steps.json:
```diff
@@ -0,0 +1,651 @@
+                "earl:Assertion"
+            ]
+        },
+        {
+            "@id": "_:sc_3.2.3",
+            "earl:test": "WCAG21:consistent-navigation(Level AA)",
+            "earl:subject": "_:subject",
+            "earl:outcome": "earl:inapplicable",
+            "dct:description": "As defined",
+            "earl:mode": "earl:semiAuto",
+            "@type": [
+                "earl:TestResult",
+                "earl:Assertion"
+            ]
+        },
```
```suggestion
        {
            "@id": "_:sc_3.2.3",
            "earl:test": "WCAG21:consistent-navigation",
            "earl:subject": "_:subject",
            "earl:outcome": "earl:inapplicable",
            "dct:description": "As defined",
            "earl:mode": "earl:manual",
            "@type": [
                "earl:TestResult",
                "earl:Assertion"
            ]
        },
```

ouafaaetta commented on méli-mélo/2021-05-steps/a11y-1-steps.json:
```diff
@@ -0,0 +1,651 @@
+                "earl:Assertion"
+            ]
+        },
+        {
+            "@id": "_:sc_3.2.4",
+            "earl:test": "WCAG21:consistent-identification(Level AA)",
+            "earl:subject": "_:subject",
+            "earl:outcome": "earl:inapplicable",
+            "dct:description": "As defined",
+            "earl:mode": "earl:semiAuto",
+            "@type": [
+                "earl:TestResult",
+                "earl:Assertion"
+            ]
+        },
```
```suggestion
        {
            "@id": "_:sc_3.2.4",
            "earl:test": "WCAG21:consistent-identification",
            "earl:subject": "_:subject",
            "earl:outcome": "earl:inapplicable",
            "dct:description": "As defined",
            "earl:mode": "earl:manual",
            "@type": [
                "earl:TestResult",
                "earl:Assertion"
            ]
        },
```

ouafaaetta commented on méli-mélo/2021-05-steps/a11y-1-steps.json:
```diff
@@ -0,0 +1,651 @@
+                "earl:Assertion"
+            ]
+        },
+        {
+            "@id": "_:sc_3.3.1",
+            "earl:test": "WCAG21:error-identification(Level A)",
+            "earl:subject": "_:subject",
+            "earl:outcome": "earl:inapplicable",
+            "dct:description": "As defined",
+            "earl:mode": "earl:semiAuto",
+            "@type": [
+                "earl:TestResult",
+                "earl:Assertion"
+            ]
+        },
```
```suggestion
        {
            "@id": "_:sc_3.3.1",
            "earl:test": "WCAG21:error-identification",
            "earl:subject": "_:subject",
            "earl:outcome": "earl:inapplicable",
            "dct:description": "As defined",
            "earl:mode": "earl:manual",
            "@type": [
                "earl:TestResult",
                "earl:Assertion"
            ]
        },
```

ouafaaetta commented on méli-mélo/2021-05-steps/a11y-1-steps.json:
```diff
@@ -0,0 +1,651 @@
+                "earl:Assertion"
+            ]
+        },
+        {
+            "@id": "_:sc_3.3.2",
+            "earl:test": "WCAG21:labels-or-instructions(Level A)",
+            "earl:subject": "_:subject",
+            "earl:outcome": "earl:inapplicable",
+            "dct:description": "As defined",
+            "earl:mode": "earl:semiAuto",
+            "@type": [
+                "earl:TestResult",
+                "earl:Assertion"
+            ]
+        },
```
```suggestion
        {
            "@id": "_:sc_3.3.2",
            "earl:test": "WCAG21:labels-or-instructions",
            "earl:subject": "_:subject",
            "earl:outcome": "earl:inapplicable",
            "dct:description": "As defined",
            "earl:mode": "earl:manual",
            "@type": [
                "earl:TestResult",
                "earl:Assertion"
            ]
        },
```

ouafaaetta commented on méli-mélo/2021-05-steps/a11y-1-steps.json:
```diff
@@ -0,0 +1,651 @@
+                "earl:Assertion"
+            ]
+        },
+        {
+            "@id": "_:sc_3.3.3",
+            "earl:test": "WCAG21:error-suggestion(Level AA)",
+            "earl:subject": "_:subject",
+            "earl:outcome": "earl:inapplicable",
+            "dct:description": "As defined",
+            "earl:mode": "earl:semiAuto",
+            "@type": [
+                "earl:TestResult",
+                "earl:Assertion"
+            ]
+        },
```
```suggestion
        {
            "@id": "_:sc_3.3.3",
            "earl:test": "WCAG21:error-suggestion",
            "earl:subject": "_:subject",
            "earl:outcome": "earl:inapplicable",
            "dct:description": "As defined",
            "earl:mode": "earl:manual",
            "@type": [
                "earl:TestResult",
                "earl:Assertion"
            ]
        },
```

ouafaaetta commented on méli-mélo/2021-05-steps/a11y-1-steps.json:
```diff
@@ -0,0 +1,651 @@
+                "earl:Assertion"
+            ]
+        },
+        {
+            "@id": "_:sc_3.3.4",
+            "earl:test": "WCAG21:error-prevention-legal-financial-data)(Level AA)",
+            "earl:subject": "_:subject",
+            "earl:outcome": "earl:inapplicable",
+            "dct:description": "As defined",
+            "earl:mode": "earl:semiAuto",
+            "@type": [
+                "earl:TestResult",
+                "earl:Assertion"
+            ]
+        },
```
```suggestion
        {
            "@id": "_:sc_3.3.4",
            "earl:test": "WCAG21:error-prevention-legal-financial-data",
            "earl:subject": "_:subject",
            "earl:outcome": "earl:inapplicable",
            "dct:description": "As defined",
            "earl:mode": "earl:manual",
            "@type": [
                "earl:TestResult",
                "earl:Assertion"
            ]
        },
```

ouafaaetta commented on méli-mélo/2021-05-steps/a11y-1-steps.json:
```diff
@@ -0,0 +1,651 @@
+                "earl:Assertion"
+            ]
+        },
+        {
+            "@id": "_:sc_4.1.1",
+            "earl:test": "WCAG21:parsing(Level A)",
+            "earl:subject": "_:subject",
+            "earl:outcome": "earl:Pass",
+            "dct:description": "Simple decoration of existing ordered lists",
+            "earl:mode": "earl:semiAuto",
+            "@type": [
+                "earl:TestResult",
+                "earl:Assertion"
+            ]
+        },
```
```suggestion
        {
            "@id": "_:sc_4.1.1",
            "earl:test": "WCAG21:parsing",
            "earl:subject": "_:subject",
            "earl:outcome": "earl:Pass",
            "dct:description": "Simple decoration of existing ordered lists",
            "earl:mode": "earl:manual",
            "@type": [
                "earl:TestResult",
                "earl:Assertion"
            ]
        },
```

ouafaaetta commented on méli-mélo/2021-05-steps/a11y-1-steps.json:
```diff
@@ -0,0 +1,651 @@
+                "earl:Assertion"
+            ]
+        },
+        {
+            "@id": "_:sc_4.1.2",
+            "earl:test": "WCAG21:Name, Role, Value (Level A)",
+            "earl:subject": "_:subject",
+            "earl:outcome": "earl:inapplicable",
+            "dct:description": "As defined",
+            "earl:mode": "earl:semiAuto",
+            "@type": [
+                "earl:TestResult",
+                "earl:Assertion"
+            ]
+        },
```
```suggestion
        {
            "@id": "_:sc_4.1.2",
            "earl:test": "WCAG21:Name, Role, Value",
            "earl:subject": "_:subject",
            "earl:outcome": "earl:inapplicable",
            "dct:description": "As defined",
            "earl:mode": "earl:manual",
            "@type": [
                "earl:TestResult",
                "earl:Assertion"
            ]
        },
```

ouafaaetta commented on méli-mélo/2021-05-steps/a11y-1-steps.json:
```diff
@@ -0,0 +1,651 @@
+                "earl:Assertion"
+            ]
+        },
+        {
+            "@id": "_:sc_4.1.3",
+            "earl:test": "WCAG21:status-messages(Level AA)",
+            "earl:subject": "_:subject",
+            "earl:outcome": "earl:inapplicable",
+            "dct:description": "As defined",
+            "earl:mode": "earl:semiAuto",
+            "@type": [
+                "earl:TestResult",
+                "earl:Assertion"
+            ]
+        }
```
```suggestion
        {
            "@id": "_:sc_4.1.3",
            "earl:test": "WCAG21:status-messages",
            "earl:subject": "_:subject",
            "earl:outcome": "earl:inapplicable",
            "dct:description": "As defined",
            "earl:mode": "earl:manual",
            "@type": [
                "earl:TestResult",
                "earl:Assertion"
            ]
        }
```

duboisp reviewed (approved):
Reviewed and locally tested. Some typos has been also fixed and the implementation plan updated.
This experimental project is now ready for being stabilized in GCWeb. We will perform a full review soon.

Christopher-O commented:
Changes in the PR

- Reduced excessive spacing for small amount of content
- Removed background color for default list steps number (except in striped mode) as was not showing up properly in high contrast mode
- Updated governance towards stabilization 

Impact for department

- No change from a departmental perspective

Impact for the public

- Created a cleaner look for items using list steps with a smaller amount of content
- Made the numbers more visible for edge case scenarios of print and high contrast

Christopher-O commented:
A few minor to dos (will create new PR): 

- Fix print issue (some text can be cut off in pattern between pages)
- Try and resolve edge case high contrast mode for Windows FF and MacOS browsers
